asset-properties-widget/component/asset-properties-config/asset-property-selector/asset-property-selector.component.ts

```diff
@@ -154,10 +155,10 @@ export class AssetPropertiesSelectorComponent {
       if(removeIndex >= 0){
         this.properties.splice(removeIndex, 1);
         property['active'] = false;
-        if(this.properties.length < 1){
-          this.contextDashboardService.formDisabled = true;
-        }else if(this.asset && this.properties.every(({ active }) => active)) {
-          this.contextDashboardService.formDisabled = false;
+        if(this.properties.length > 0 && this.properties.every(({ active }) => active)){
+          this.isPropertySelected = true;
+        }else {
+          this.isPropertySelected = false;
```

Comment:
why is every used to check if atleast one property is selected or not?

Reply:
every remove I am checking is atleast one property is exist or not and that flag used to show worning alert.